hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ContainerReplicaOp.java

```diff
@@ -36,6 +36,12 @@ public enum PendingOpType {
   private int replicaIndex;
   private long scheduledEpochMillis;
 
+  public static ContainerReplicaOp create(PendingOpType opType,
+      DatanodeDetails target, int replicaIndex) {
+    return new ContainerReplicaOp(opType, target, replicaIndex,
+        System.currentTimeMillis());
+  }
```

Comment:
Nit: as far as I see this is only for test (`TestECContainerHealthCheck`), wouldn't it be better to define as helper in that class (or `ReplicationTestUtil`)?

Reply:
So far, we don't create any ContainerRepliaOps except in tests, but the main code will soon need to create them, and this static method should make it a little bit nicer to create them in the main code flow when we do. What do you think?

Reply:
We can keep it this way then.